### DISCUSSION TITLE
Local file access

### DISCUSSION THREAD

**Discussion by gdiana** [grafana/grafana-infinity-datasource#129] [Ideas]
Hi,

Is it at all possible to allow local file access. Currently, URL or inline cut and paste is allowed but it would be useful to allow direct access to local files. (c:/.......)

I am not sure if this is a mission to do but would be useful. In my case, I use node-red and other s/w to do pre-data processing and some only write to their own program directory and not to the root IIS directory.

Kind regards

Greg Diana

**Comment by Darecki1** [comment]
In windows you can add IIS role and then enable directory browsing and access file using http://localhost/file.xml path.

**Reply by gdiana** [reply]
Hi, 
Yes I am actually doing that but it is not working 100%. Works 100% when used in browser but not from within Grafana. Sometimes works sometimes does not using infinity csv as source

**Reply by gdiana** [reply]
Ha, I found my csv files were corrupted so works both from browser, and within GRAFANA
